CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,81 @@
+# PureScript Community Code of Conduct
+
+The PureScript community follows some simple ground rules to protect safe, inclusive spaces to discuss the PureScript programming language.
+
+All community members are expected to act and interact in ways that contribute to an open, welcoming, diverse, inclusive, and healthy community.
+
+## What's our purpose?
+
+The PureScript community channels are places to discuss the PureScript programming language. 
+
+* We talk about the language, its related libraries and infrastructure, and our experiences using it.
+* We help PureScript programmers learn the language, and we ask for help becoming stronger programmers ourselves. 
+* We discuss general functional programming concepts and other parts of programming, type theory, and mathematics that are relevant to Purescript.
+* We organize open-source work, meetups, hackathons, and other community events.
+* We discuss what we can do to grow an inclusive, friendly, respectful community.
+
+## What is okay?
+
+Examples of behavior that contributes to a positive environment for our community include:
+
+* Demonstrating empathy and kindness toward other people.
+* Being respectful of differing opinions, viewpoints, and experiences.
+* Giving and gracefully accepting constructive feedback.
+* Accepting responsibility and apologizing to those affected by our mistakes, and learning from the experience.
+
+As participants in the PureScript community, we pledge to make participation in our community a harassment-free experience for everyone -- regardless of age, body size, disability, ethnicity, sex characteristics, gender identity and expression, level of experience, education, socio-economic status, nationality, personal appearance, race, caste, color, religion, or sexual identity and orientation.
+
+## What is not okay?
+
+Examples of unacceptable behavior include:
+
+* The use of sexualized language or imagery, and sexual attention or advances of any kind.
+* Trolling, insulting or derogatory comments, and personal or political attacks.
+* Public or private harassment.
+* Publishing others’ private information, such as a physical or email address, without their explicit permission.
+* Other conduct which could reasonably be considered inappropriate in a professional setting.
+
+
```

Comment:
Is there a thematic breakup here to have 2 newlines? Typo or `---`?

Reply:
This was unintentional -- it's just meant to be one newline.